gems/aws-sdk-core/lib/aws-sdk-core/plugins/regional_endpoint.rb

```diff
@@ -81,6 +99,13 @@ def resolve_region(cfg)
           env_region || cfg_region
         end
 
+        def resolve_sigv4a_signing_region_set(cfg)
+          value = ENV['AWS_SIGV4A_SIGNING_REGION_SET']
+          value ||= Aws.shared_config.sigv4a_signing_region_set(profile: cfg.profile)
+          value ||= cfg.region || ''
```

Comment:
I think this shouldn't default to cfg.region - I think the config should be an override and most of the time should be unset.

Reply:
Yes. This is old code.